docs/ref/settings.txt

```diff
@@ -687,13 +689,16 @@ When :setting:`USE_TZ` is ``False``, it is an error to set this option.
     third-party systems connect to the same database and expect to find
     datetimes in local time, then you must set this option.
 
-* If the database backend supports time zones (e.g. PostgreSQL), the
-  ``TIME_ZONE`` option is very rarely needed. It can be changed at any time;
-  the database takes care of converting datetimes to the desired time zone.
+* If the database backend supports time zones (e.g. PostgreSQL) then the
+  database connection's time zone set to this value.
```

Comment:
```suggestion
* If the database backend supports time zones (e.g., PostgreSQL), then the
  database connection's time zone is set to this value.
```

TIL.
e.g. meaning for example is always followed by a comma.

Reply:
Thanks missed a couple of other bits too 👍